### PR TITLE
AT_16.01_009 | <Multibranch Pipeline> Fill out and verify configuration> General fields

### PR DESCRIPTION
**Pull request by larysareznikova**
https://trello.com/c/Jy4i7iZ2/1247-at1601009-multibranch-pipeline-fill-out-and-verify-configuration-general-fields

### CODE REVIEW

**Review by Gulya111 — APPROVED** [review]

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const displayName = 'displayName' + Date.now()
+
+    beforeEach('Create multibranch pipeline', function () {
+        createMultiBranchPipeline(newPipelineName);
+        cy.get('a[href="./configure"]').click();
+    })
+
+    it('AT 16.01.009|fill out and verify multibranch pipeline configuration General fields', function () {
+        cy.get('.jenkins-form-item > .jenkins-form-label')
+            .should('contain', 'Display Name')
```
Comment (Rom-T) [inline cypress/e2e/multiBranchPipelineConfiguration.cy.js:16]: Please move `Display Name` in the fixtures

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        cy.get('a[title="Help for feature: Display Name"]')
+            .click()
+        cy.get('.jenkins-form-item > .help-area > .help > :nth-child(1)')
+            .should('be.visible')
+        cy.get('.jenkins-form-item > .jenkins-form-label')
+            .should('contain', 'Description')
```
Comment (Rom-T) [inline cypress/e2e/multiBranchPipelineConfiguration.cy.js:22]: Please move `Description` in the fixtures

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            .realHover()
+            .should('be.visible')
+        cy.get('input[name="_.displayNameOrNull"]')
+            .should('be.visible')
+            .type(displayName)
+        cy.get('textarea[name="_.description"]')
+            .should('be.visible')
+            .type(description)
+        cy.get('.textarea-show-preview')
+            .should('contain', 'Preview')
```
Comment (Rom-T) [inline cypress/e2e/multiBranchPipelineConfiguration.cy.js:33]: Please move `Preview` in the fixtures

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            .click()
+        cy.get('.textarea-preview').should('contain', description)
+        cy.get('.textarea-hide-preview')
+            .should('be.visible')
+            .and('contain', 'Hide preview')
```
Comment (Rom-T) [inline cypress/e2e/multiBranchPipelineConfiguration.cy.js:38]: Please move `Hide preview` in the fixtures

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js (hunk 1 above)
Comment (larysareznikova) [reply cypress/e2e/multiBranchPipelineConfiguration.cy.js]: done

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js (hunk 3 above)
Comment (larysareznikova) [reply cypress/e2e/multiBranchPipelineConfiguration.cy.js]: done

File: cypress/e2e/multiBranchPipelineConfiguration.cy.js (hunk 2 above)
Comment (larysareznikova) [reply cypress/e2e/multiBranchPipelineConfiguration.cy.js]: done

**Review by RimmaSalikh — APPROVED** [review]

**Review by Gulya111 — APPROVED** [review]

**Pull request merged**